### PR TITLE
Dwf 2017 11 17 jenkins

### PR DESCRIPTION
**Pull request by kurtseifried**
reserved stuff

### CODE REVIEW

**Comment by cve-team** [comment]
This pull request covers CVE-2017-15100 as well as #94 (and #95). And like those other PRs, the description is out of sync with the references.

File: 2017/15xxx/CVE-2017-15100.json [hunk 1]
```diff
@@ -1,8 +1,31 @@
 {
```
Comment (cve-team) [inline 2017/15xxx/CVE-2017-15100.json]: JSON: Expecting property name: line 8 column 4 (char 205)

**Comment by cve-team** [comment]
Auto-validation for this pull request has detected issues:
JSON: Please see review comments for errors.
JSON: Please see review comments for errors.
JSON: Please see review comments for errors.
JSON: Please see review comments for errors.
JSON: Please see review comments for errors.
JSON: Please see review comments for errors.
JSON: Please see review comments for errors.
JSON: Please see review comments for errors.
JSON: Please see review comments for errors.
JSON: Please see review comments for errors.
JSON: Please see review comments for errors.
JSON: Please see review comments for errors.
JSON: Please see review comments for errors.
JSON: Please see review comments for errors.
JSON: Please see review comments for errors.
JSON: Please see review comments for errors.
JSON: Please see review comments for errors.
JSON: Please see review comments for errors.
JSON: Please see review comments for errors.

File: 2017/1000xxx/CVE-2017-1000386.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000386.json]: JSON: Expecting property name: line 8 column 4 (char 205)

File: 2017/1000xxx/CVE-2017-1000387.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000387.json]: JSON: Expecting property name: line 8 column 4 (char 205)

File: 2017/1000xxx/CVE-2017-1000388.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000388.json]: JSON: Expecting property name: line 8 column 4 (char 205)

File: 2017/1000xxx/CVE-2017-1000389.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000389.json]: JSON: Expecting property name: line 8 column 4 (char 205)

File: 2017/1000xxx/CVE-2017-1000390.json [hunk 6]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000390.json]: JSON: Expecting property name: line 8 column 4 (char 205)

File: 2017/1000xxx/CVE-2017-1000391.json [hunk 7]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000391.json]: JSON: Expecting property name: line 8 column 4 (char 205)

File: 2017/1000xxx/CVE-2017-1000392.json [hunk 8]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000392.json]: JSON: Expecting property name: line 8 column 4 (char 205)

File: 2017/1000xxx/CVE-2017-1000393.json [hunk 9]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000393.json]: JSON: Expecting property name: line 8 column 4 (char 205)

File: 2017/1000xxx/CVE-2017-1000394.json [hunk 10]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000394.json]: JSON: Expecting property name: line 8 column 4 (char 205)

File: 2017/1000xxx/CVE-2017-1000395.json [hunk 11]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000395.json]: JSON: Expecting property name: line 8 column 4 (char 205)

File: 2017/1000xxx/CVE-2017-1000396.json [hunk 12]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000396.json]: JSON: Expecting property name: line 8 column 4 (char 205)

File: 2017/1000xxx/CVE-2017-1000397.json [hunk 13]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000397.json]: JSON: Expecting property name: line 8 column 4 (char 205)

File: 2017/1000xxx/CVE-2017-1000398.json [hunk 14]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000398.json]: JSON: Expecting property name: line 8 column 4 (char 205)

File: 2017/1000xxx/CVE-2017-1000399.json [hunk 15]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000399.json]: JSON: Expecting property name: line 8 column 4 (char 205)

File: 2017/1000xxx/CVE-2017-1000400.json [hunk 16]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000400.json]: JSON: Expecting property name: line 8 column 4 (char 205)

File: 2017/1000xxx/CVE-2017-1000401.json [hunk 17]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000401.json]: JSON: Expecting property name: line 8 column 4 (char 205)

File: 2017/1000xxx/CVE-2017-1000402.json [hunk 18]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000402.json]: JSON: Expecting property name: line 8 column 4 (char 205)

File: 2017/1000xxx/CVE-2017-1000403.json [hunk 19]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000403.json]: JSON: Expecting property name: line 8 column 4 (char 205)

File: 2017/1000xxx/CVE-2017-1000404.json [hunk 20]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000404.json]: JSON: Expecting property name: line 8 column 4 (char 205)

**Comment by cve-team** [comment]
Auto-validation for this pull request has detected issues:
JSON: Please see review comments for errors.

**Comment by kurtseifried** [comment]
What can I do from my side to get this commit submitted? thanks.

**Comment by cve-team** [comment]
The validation problems occur because of the trailing comma after the STATE string. Removing those should fix those.

The PR also includes CVE-2017-15100, which was assigned by Red Hat and uses a description that is out of line with the references. I recommend replacing 2017/15xxx/CVE-2017-15100.json with what is now in "master", which effectively removes the update to it from this PR.

George

**Pull request closed without merging**